docs/anthropic.md

```diff
@@ -9,21 +9,11 @@ This page shows you how to:
 
 Anthropic functions in pgai require an [Anthropic API key](https://docs.anthropic.com/en/docs/quickstart#set-your-api-key).
 
-- [Handle API keys using pgai from psql](#handle-api-keys-using-pgai-from-psql)
-- [Handle API keys using pgai from python](#handle-api-keys-using-pgai-from-python)
+The simplest option for interactive use is to configure the API key as a
```

Comment:
not sure about this text since the session level parameter isn't the recommended method. Let's think.

Reply:
What do you propose?

Reply:
```suggestion
In production, we suggest setting the API key using an environment variable. During testing and development, it may be easiest to pass the key value in directly using the `api_key` parameter.  For more options and details, consult the
[Handling API keys](./handling-api-keys.md) document.
```

Reply:
I'm not really happy with this suggestion. I agree with telling users to use an env var as first prio. I don't think that we should suggest passing the key value directly (instead of the session-level parameter). The main reason I don't like it is that it will definitely result in the secret being exposed in the logs.